Aztec/Classes/TextKit/ParagraphStyle.swift

```diff
@@ -214,26 +188,42 @@ open class ParagraphStyle: NSMutableParagraphStyle, CustomReflectable {
     var baseHeadIndent: CGFloat = 0
     var baseFirstLineHeadIndent: CGFloat = 0
     var baseTailIndent: CGFloat = 0
-    var baseParagraphSpacing: CGFloat = 0
-    var baseParagraphSpacingBefore: CGFloat = 0
-
-    open override class var `default`: NSParagraphStyle {
+    
+    var textListParagraphSpacing = CGFloat(0)
+    var textListParagraphSpacingBefore = CGFloat(0)
+    
+    // MARK: - Defaults
+    
+    private static var cachedDefault: ParagraphStyle = {
         let style = ParagraphStyle()
-
+        
         var tabStops = [NSTextTab]()
-
+        
         for intervalNumber in (1 ..< Metrics.tabStepCount) {
             let location = intervalNumber * Metrics.tabStepInterval
             let textTab = NSTextTab(textAlignment: .natural, location: CGFloat(location), options: [:])
-
+            
             tabStops.append(textTab)
         }
-
+        
         style.tabStops = tabStops
-
+        style.lineSpacing = 8
+        style.paragraphSpacing = 8
+        style.paragraphSpacingBefore = 8
+        
         return style
+    }()
+    
+    static func setDefault(_ newDefault: ParagraphStyle) {
+        cachedDefault = newDefault
```

Comment:
So we can now configure the default here?

Reply:
That's the idea... this may change over time though, but that's the initial hook we will be using.